src/BaGet.Web/Controllers/IndexController.cs

```diff
@@ -10,31 +12,33 @@ namespace BaGet.Web.Controllers
     /// </summary>
     public class IndexController : Controller
     {
+        private IEnumerable<ServiceResource> ServiceWithAliases(string name, string url, params string[] versions) {

```

Comment:
Let's stick to .NET Foundation's style guide (see [this](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)):

1. `{` on new lines
2. Spaces between `foreach`/`if` and `(`